src/config/config.go

```diff
@@ -54,6 +54,8 @@ const (
 	ZarfInClusterContainerRegistryNodePort = 31999
 
 	ZarfInClusterGitServiceURL = "http://zarf-gitea-http.zarf.svc.cluster.local:3000"
+
+	ZarfSeedImage = "registry:2.8.1"
```

Comment:
This feels like a weird thing to code into the binary. I see that you're doing this since you don't want to have the `seed` section for every package when it is only needed for initialization packages and I don't have any immediate suggestions but this feels off since it forces us to build a new Zarf binary if we want to update the registry component with a new container version.

Reply:
We could change it to an env variable--but they've just started any useful updates again, it remained static for a couple years.  This will update probably every 2-3 minor bumps of zarf (every 3-4 months)

Reply:
Maybe one day if this still bothers me in the future I can look into some form of env variable (or specialized Zarf Package Variable). But I'm fine with the way it is for now.